### ISSUE TITLE
Remove `disabled` state from all interactive components

### ISSUE THREAD

**Issue by ppvg** [minvws/nl-rdo-manon#950]
The use of the `disabled` attribute, or any other way to "disable" an interactive element, is harmful for accessibility. Instead of being disabled, the element should remain functional and provide the user with clear feedback about why it cannot be operated (yet).

### Acceptance criteria
- [ ] All styling for `:disabled` selectors (or similar) is removed.
- [ ] All examples of using the `disabled` attribute (or similar) are removed, except those used to illustrate what **not** to do.
- [ ] The documentation for components that used to support a "disabled" state, now provide an explanation about why not to use it and suggestions for what to do instead.

**Comment by HeleenSG** [comment]
Fixed for v18: https://github.com/minvws/nl-rdo-manon/pull/1081
